client/lib/google-apps/index.js

```diff
@@ -1,26 +1,19 @@
 /**
- * External dependencies
+ * Internal dependencies
  */
-import { numberFormat } from 'i18n-calypso';
+import formatCurrency from 'lib/format-currency';
 
 /**
  * Constants
  */
 const GOOGLE_APPS_LINK_PREFIX = 'https://mail.google.com/a/';
-const PRICE_FORMAT = /(\d+[.,]?\d+)/;
 
-function getAnnualPrice( price ) {
-	return price && price.replace( PRICE_FORMAT, ( value ) => {
-		const number = parseFloat( value );
-		return number % 1 === 0 ? number : numberFormat( number, 2 );
-	} );
+function getAnnualPrice( cost, currencyCode ) {
+	return formatCurrency( cost, currencyCode, { precision: 0 } );
```

Comment:
That was based on the original design - and with the old currencies, none provided decimal points. But I didn't like it either, so I'll change the precision to the default one.